modules/aruco/misc/python/pattern_generator/README.md

```diff
@@ -0,0 +1,42 @@
+# OpenCVMarkerPrinter
+
+## Description
+This small app can save some commonly used opencv markers such as ArUco, ArUcoGrid, Chessboard and ChArUco to vector graphics file. **Supported vector graphics file format: .svg, .pdf and .ps. Supported image file format: .png (not recommend).**
+
+<img src="./doc/images/0001.jpg" height="160" />
+
+### Dependencies
+#### MarkerPrinter
+  * numpy
+  * opencv-python
+  * opencv-contrib-python
```

Comment:
`opencv-python` and `opencv-contrib-python` are external (unofficial) distributions.

Reply:
@alalek OK~ I will remove the dependency.
By the way, do I need to take care of the licence of imported libraries?
For example, CairoSVG use LGPLv3 and Cairo use LGPLv2, MPL 1.1.

Reply:
I believe, LGPL is fine.